cmd/virt-launcher-monitor/virt-launcher-monitor.go

```diff
@@ -197,11 +198,12 @@ func RunAndMonitor(containerDiskDir, uid string) (int, error) {
 
 				log.Log.Infof("Reaped pid %d with status %d", wpid, int(wstatus))
 				if wpid == cmd.Process.Pid {
+					log.Log.Infof("Reaped Launcher main pid")
```

Comment:
Maybe we could move the log above, below this if statement, wdyt?

Reply:
Just to make sure i understand you suggest making it look like this:

```
				if wpid == cmd.Process.Pid {
					log.Log.Infof("Reaped Launcher main pid")
					log.Log.Infof("Reaped pid %d with status %d", wpid, int(wstatus))

```

If so why?

Reply:
I meant like this
```
if wpid == cmd.Process.Pid {
	log.Log.Infof("Reaped Launcher main pid")
        exitStatus <- wstatus.ExitStatus()
}
log.Log.Infof("Reaped pid %d with status %d", wpid, int(wstatus))
```
but it is really nit

Reply:
Done